airbyte-integrations/connectors/source-google-analytics-data-api/unit_tests/test_streams.py

```diff
@@ -127,7 +127,7 @@ def test_parse_response(patch_base_class):
             {"name": "totalUsers", "type": "TYPE_INTEGER"},
             {"name": "newUsers", "type": "TYPE_INTEGER"},
             {"name": "sessions", "type": "TYPE_INTEGER"},
-            {"name": "sessionsPerUser:", "type": "TYPE_FLOAT"},
+            {"name": "sessionsPerUser:parameter", "type": "TYPE_FLOAT"},
```

Comment:
Why does it now get suffixed by `parameter`? This doesn't seem related to the switch from `_` to `:`

Reply:
Yes, but custom dimentions structure is `<scope>:<parameter>`, so I deside to add parameter after `:`

Reply:
Is this a breaking change for current users that would have not set up the source after the breaking change as well? If feels like a field is being removed and another one added

Reply:
Should be if they not use normalization in other way the get an error during normalisation.